changes/4171.added

```diff
@@ -0,0 +1 @@
+Added missing DelineaSecretChoices TYPE_NOTES and TYPE_URL to SecretsGroupSecretTypeChoices
```

Comment:
I am not super familiar with `DelineaSecretChoices` but I think perhaps stating it this way might be more generic and more understandable.

```suggestion
Added `TYPE_NOTES` and `TYPE_URL` to SecretsGroupSecretTypeChoices.
```